tests/toitp/CMakeLists.txt

```diff
@@ -42,21 +44,26 @@ foreach(file ${TOITP_TESTS})
   add_dependencies(check_slow ${build_snap})
 
   file(RELATIVE_PATH toitp_test_name ${TOIT_SDK_SOURCE_DIR} ${toitp_test})
-  add_test(
-    NAME ${toitp_test_name}
-    COMMAND $<TARGET_FILE:toit.run> ${toitp_test} ${snap} $<TARGET_FILE:toit.run> ${TOIT_SDK_SOURCE_DIR}/tools/toitp.toit
-    )
-  set_tests_properties(${toitp_test_name} PROPERTIES TIMEOUT 40)
+  if (NOT "${toitp_test_name}" IN_LIST TOIT_SKIP_TESTS)
+    add_test(
+      NAME ${toitp_test_name}
+      COMMAND $<TARGET_FILE:toit.run> ${toitp_test} ${snap} $<TARGET_FILE:toit.run> ${TOIT_SDK_SOURCE_DIR}/tools/toitp.toit
+      )
+    set_tests_properties(${toitp_test_name} PROPERTIES TIMEOUT 40)
+    if ("${toitp_test_name}" IN_LIST TOIT_FAILING_TESTS)
+      set_tests_properties(${toitp_test_name} PROPERTIES WILL_FAIL TRUE)
+    endif()
+  endif()
 
   file(RELATIVE_PATH snap_test_name ${TOIT_SDK_SOURCE_DIR} ${snap_test})
-  add_test(
-    NAME ${snap_test_name}
-    COMMAND $<TARGET_FILE:toit.run> ${snap_test} ${snap}
-    )
-  set_tests_properties(${snap_test_name} PROPERTIES TIMEOUT 40)
-endforeach()
-
-include(fail.cmake)
-foreach(test ${TOIT_FAILING_TESTS})
-  set_tests_properties(${test} PROPERTIES WILL_FAIL TRUE)
+  if (NOT "${snap_test_name}" IN_LIST TOIT_SKIP_TESTS)
```

Comment:
continue for skipping?

Reply:
Since the toitp_test is in an `if` and not with `continue` I feel it's more consistent to do the same for the snap test.